resources/factions/bluefor_modern_2015.json

```diff
@@ -0,0 +1,142 @@
+{
+  "country": "Combined Joint Task Forces Blue",
+  "name": "Bluefor Modern 2015",
```

Comment:
Why a new faction? And this one is formatted quite wrong. A lot of indents are not right.